samples/bluetooth/iso_receive/src/main.c

```diff
@@ -215,12 +217,18 @@ static void iso_recv(struct bt_iso_chan *chan, const struct bt_iso_recv_info *in
 
 	if (buf->len == sizeof(count)) {
 		count = sys_get_le32(buf->data);
+		if (IS_ENABLED(CONFIG_ISO_ALIGN_PRINT_INTERVALS)) {
+			iso_recv_count = count;
+		}
 	}
 
-	str_len = bin2hex(buf->data, buf->len, data_str, sizeof(data_str));
-	printk("Incoming data channel %p flags 0x%x seq_num %u ts %u len %u: "
-	       "%s (counter value %u)\n", chan, info->flags, info->seq_num,
-	       info->ts, buf->len, data_str, count);
+	if ((iso_recv_count % CONFIG_ISO_PRINT_INTERVAL) == 0) {
+		str_len = bin2hex(buf->data, buf->len, data_str, sizeof(data_str));
+		printk("Incoming data channel %p flags 0x%x seq_num %u ts %u len %u: "
+		       "%s (counter value %u)\n", chan, info->flags, info->seq_num,
+		       info->ts, buf->len, data_str, count);
+	}
+	iso_recv_count++;
```

Comment:
We usually keep an empty line after ending brackets
```suggestion
	}
	
	iso_recv_count++;
```

Reply:
Done